src/test/java/uk/nhs/digital/nhsconnect/lab/results/mesh/RecipientMailboxIdMappingsTest.java

```diff
@@ -21,35 +19,14 @@ void setUp() {
 
     @Test
     void testGetRecipientMailboxIdForMessage() {
-
-        final OutboundMeshMessage message = new MeshMessage().setRecipient("REC1");
-
-        assertEquals("test_mailbox", recipientMailboxIdMappings.getRecipientMailboxId(message));
+        assertEquals("test_mailbox", recipientMailboxIdMappings.getRecipientMailboxId("REC1"));
     }
 
     @Test
     void testGetRecipientMailboxIdForMessageRecipientNotFoundThrowsException() {
-
-        final OutboundMeshMessage message = new MeshMessage().setRecipient("INVALID");
-
         final MeshRecipientUnknownException exception = assertThrows(MeshRecipientUnknownException.class,
-            () -> recipientMailboxIdMappings.getRecipientMailboxId(message));
+            () -> recipientMailboxIdMappings.getRecipientMailboxId("INVALID"));
 
         assertEquals("Couldn't decode recipient: INVALID", exception.getMessage());
     }
-
-    @Test
-    void testGetRecipientMailboxIdForMessageNoRecipientToMailboxMappingsThrowsException() {
-
-        recipientMailboxIdMappings = new RecipientMailboxIdMappings("");
-
-        final OutboundMeshMessage message = new MeshMessage();
-
-        final MeshRecipientUnknownException exception = assertThrows(MeshRecipientUnknownException.class,
-            () -> recipientMailboxIdMappings.getRecipientMailboxId(message));
```

Comment:
This test scenario could still be valid if an empty string was passed as the recipient